Lib/idlelib/idle_test/test_query.py

```diff
@@ -137,8 +137,11 @@ def test_good_module_name(self):
         self.assertTrue(dialog.entry_ok().endswith('__init__.py'))
         self.assertEqual(dialog.entry_error['text'], '')
         dialog = self.Dummy_ModuleName('os.path')
```

Comment:
os.path is an arbitrary dotted module name.  I would rather switch.
Change is in two pieces because github would not do it in one.
```suggestion
        dialog = self.Dummy_ModuleName('idlelib.idle')
```

Reply:
I've made this change.  Thanks!